test/examples/test_kissgp_additive_regression.py

```diff
@@ -14,7 +14,7 @@
 from torch.autograd import Variable
```

Comment:
Variable is deprecated - would you mind mind removing it from this example while you're working on this?